qpc/cred/add.py

```diff
@@ -47,21 +47,26 @@ def __init__(self, subparsers):
         self.parser.add_argument('--type', dest='type',
                                  choices=[credential.NETWORK_CRED_TYPE,
                                           credential.VCENTER_CRED_TYPE,
-                                          credential.SATELLITE_CRED_TYPE],
+                                          credential.SATELLITE_CRED_TYPE,
+                                          credential.OPENSHIFT_CRED_TYPE],
                                  metavar='TYPE',
                                  help=_(messages.CRED_TYPE_HELP),
+                                 type=str.lower,
                                  required=True)
         self.parser.add_argument('--username', dest='username',
                                  metavar='USERNAME',
                                  help=_(messages.CRED_USER_HELP),
-                                 required=True)
+                                 required=False)
```

Comment:
I wonder if this change won't bring unintended side effects to other credential types :thinking:

Reply:
Hm. I know that this is dealt with in the server side.  There's no function in the cli that validates the fields for each cred type.

Reply:
OK, then!